### PR TITLE
Adding Phil Stollery

### PR DESCRIPTION
**Pull request by PhilStollery**
**Add your name to verify:** Phil Stollery

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -6912,6 +6912,31 @@ module.exports = [
       'WordPress',
     ],
   },
+  {
+    name: 'Phil Stollery',
+    description: 'Technical writer, recovering developer, lover of Whippets, from Yorkshire.',
+    url: 'https://stollerys.co.uk/setup',
+    twitter: '@phils@social.lol',
```
Comment (PhilStollery) [inline src/data.js:6919]: Will this work, or does the site redirect to X?

File: src/data.js [hunk 2]
```diff
@@ -6912,6 +6912,31 @@ module.exports = [
...
+    emoji: '🥋',
+    country: '🇬🇧',
+    computer: 'apple',
+    phone: 'iphone',
+    tags: [
+      'Aikido',
```
Comment (PhilStollery) [inline src/data.js:6925]: I use my martial arts skills every day whilst coding are these allowed?

**Review by gatarelib — APPROVED** [review]

File: src/data.js (hunk 1 above)
Comment (ballpointcarrot) [reply src/data.js:6919]: Looks like this failed a validation in the PR checks: https://github.com/wesbos/awesome-uses/actions/runs/5976959754/job/16784742144?pr=1654

**Comment by wesbos** [comment]
closing as it's not a /uses page. Feel free to re-PR once changed

**Pull request closed without merging**